restcomm/restcomm.telephony/src/main/java/org/restcomm/connect/telephony/CallManager.java

```diff
@@ -1727,11 +1789,39 @@ private void outboundToPstn(final CreateCall request, final ActorRef sender) thr
 
 
     private void outboundToSip(final CreateCall request, final ActorRef sender) throws ServletParseException {
+        final String uri = (request.getOutboundProxy()!= null && (!request.getOutboundProxy().isEmpty())) ? request.getOutboundProxy() : "";
         SipURI outboundIntf = null;
         SipURI from = null;
         SipURI to = null;
 
-        to = (SipURI) sipFactory.createURI(request.to());
+        String toUser = "";
+        String toHost = "";
+        String scheme = "";
+        boolean isSecure = false;
+        if(request.to().contains("@") &&
```

Comment:
@abdulazizali77 why we need this part of the code? 
Since we reach to this point we have the following:
1. Previouly we checked and we know this is a SIP URI (contains @ in the request.to()).
2. After we create the `SipURI to = (SipURI) sipFactory.createURI(request.to());` we can check if its secure or not using the API provided method `to.isSecure();`.

If the `request.to()` contains `sips` then the resulted SipURI will be secure.